common/main/java/com/couchbase/lite/internal/support/Log.java

```diff
@@ -322,6 +323,7 @@ private static void log(
     }
 
     private static boolean shouldLog(@NonNull LogLevel logLevel) {
+        if ((LogLevel.VERBOSE.compareTo(logLevel) >= 0) && !CouchbaseLiteInternal.debugging()) { return false; }
```

Comment:
Another change that is not strictly related to the topic: Don't log below the VERBOSE level, unless debugging.  Debugging *is* configurable at runtime.